tests/target/issue-5507/issue-5507-version-two.rs

```diff
@@ -0,0 +1,35 @@
+// rustfmt-version: Two
+
+struct EmptyBody<T>
+where
+    T: Eq,
+{}
+
+struct LineComment<T>
+where
+    T: Eq,
+{
+    // body
+}
+
+struct MultiLineComment<T>
+where
+    T: Eq,
+{
+    /*
+    Multiline
+    comment.
+    */
+}
+
+struct BlockComment<T>
+where
+    T: Eq,
+{/* block comment */}
```

Comment:
Similarly, I think single line block comments should also follow the `version=One` formatting.

Reply:
Is there a way to specifically check for block comments? If so, I the solution shouldn't be too complicated.

Reply:
To my knowledge there isn't a standalone function that can be used for something like that. However you could check that using [`CommentStyle`](https://github.com/rust-lang/rustfmt/blob/38659ec6ad5f341cf8eb3139725bf695872c6de7/src/comment.rs#L42), which has a [`is_block_comment`](https://github.com/rust-lang/rustfmt/blob/38659ec6ad5f341cf8eb3139725bf695872c6de7/src/comment.rs#L73) method. You'd use the [`comment_style`](https://github.com/rust-lang/rustfmt/blob/38659ec6ad5f341cf8eb3139725bf695872c6de7/src/comment.rs#L128) function to construct one.

Reply:
@ytmimi Done! Now empty `impl` blocks and `impl` blocks that only contain single line `block comments` have the same format on each `version`.

Reply:
Fantastic. Thank you for making those changes. I'll give this another review in the coming days!